docs/source/guides/development/cli_guide.rst

```diff
@@ -48,7 +52,7 @@ Alice Grants Access To Secret
 .. code::
 
     (nucypher)$ nucypher alice grant \
-    >     --teacher gemini.nucypher.network:9151 \
+    >     --teacher <SEEDNODE URI> \
```

Comment:
The `--teacher` flag is optional with the latest version of domain and registry resolving, and removing it might make for a more simple example.

Reply:
✔️ - also done for `nucypher bob retrieve`